From 119195c0e2b0c78122bfd928e1908a1d4d919422 Mon Sep 17 00:00:00 2001
Message-Id: <119195c0e2b0c78122bfd928e1908a1d4d919422.1414996573.git.chang-joon.lee@intel.com>
In-Reply-To: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
References: <de6d11b8587be186d089e692fc410008d4d87034.1414996573.git.chang-joon.lee@intel.com>
From: Rafael Barbalho <rafael.barbalho@intel.com>
Date: Fri, 31 Oct 2014 11:36:28 +0000
Subject: [PATCH 06/10] FOR_UPSTREAM [VPG]: drm/i915: Remove unused variable
 in vlv_compute_drain_latency

The following patch didn't correctly remove all variables:

commit e6bb1f02089ba78cfbea6f49b5d225e888e14fc0
Author: Yogesh Mohan Marimuthu <yogesh.mohan.marimuthu@intel.com>
Date:   Tue Oct 14 14:59:49 2014 +0530

    FOR_UPSTREAM [VPG]: drm/i915: optimize dl for baytrail and use correct precision for cht

For: GMINL-1778
Change-Id: Ie57f5a4bff6ddf6d083d938236171f3685c44c4c
Signed-off-by: Rafael Barbalho <rafael.barbalho@intel.com>
---
 drivers/gpu/drm/i915/intel_pm.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_pm.c b/drivers/gpu/drm/i915/intel_pm.c
index 46eb7c4..630091a 100644
--- a/drivers/gpu/drm/i915/intel_pm.c
+++ b/drivers/gpu/drm/i915/intel_pm.c
@@ -1536,7 +1536,6 @@ static bool vlv_compute_drain_latency(struct drm_device *dev,
 				struct vlv_MA_component_enabled enable)
 {
 	struct drm_crtc *crtc;
-	int entries;
 	bool latencyprogrammed = false;
 
 	crtc = intel_get_crtc_for_pipe(dev, pipe);
-- 
1.7.9.5

